wolfssl/wolfcrypt/port/Renesas/renesas-sce-crypt.h

```diff
@@ -0,0 +1,231 @@
+/* renesas-sce-crypt.h
+ *
+ * Copyright (C) 2006-2021 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+#ifndef __RENESAS_SCE_CRYPT_H__
+#define __RENESAS_SCE_CRYPT_H__
+
+#include "r_sce.h"
+#include <wolfssl/wolfcrypt/settings.h>
+#include <wolfssl/wolfcrypt/logging.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define SCE_SESSIONKEY_NONCE_SIZE           8
+#define WOLFSSL_SCE_ILLEGAL_CIPHERSUITE     -1
+
+    enum {
```

Comment:
Please add a comment here stating that these are the supported cipher suites by SCE/FSP version XXX.

Reply:
Added comment